src/redux/air-pollution/airPollutionsSlice.js

```diff
@@ -0,0 +1,53 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const apiKey = 'cd559a00a031c2d7baba09afe0b27bbb';
+
+const baseUrl = 'http://api.openweathermap.org/data/2.5/air_pollution';
+
+const initialState = {
+  airPollutions: {},
+  airpollutionsLoading: false,
+  airpollutionsErros: null,
+};
+
+export const fetchAirPollutionData = createAsyncThunk('air-pollutions/fetchAirPollutionData', async (action) => {
+  try {
+    const { lat, lon } = action;
+    const res = await fetch(`${baseUrl}?lat=${lat}&lon=${lon}&appid=${apiKey}`);
+    return res.json();
+  } catch (error) {
+    return error;
+  }
+});
+
+export const airPollutionSlice = createSlice({
+  name: 'airPollution',
+  initialState,
+  extraReducers: (builder) => {
+    // Fetch metrics
+    builder.addCase(fetchAirPollutionData.pending, (state) => ({
+      ...state, airpollutionsLoading: true,
+    }));
+    builder.addCase(fetchAirPollutionData.fulfilled, (state, action) => {
+      const data = action.payload?.list[0];
```

Comment:
- [ ] There seems to be an error in this line of code. When I navigate to the details page, I cannot seem to see the fetched data. When I check the console, the error below appears ⬇️ 
![Screenshot 2023-04-20 at 10 08 02](https://user-images.githubusercontent.com/32361963/233288115-4083d3e8-a57a-4958-adac-7fee2db65668.png)

Kindly fix that error